test/grpc.ts

```diff
@@ -207,6 +207,7 @@ describe('grpc', () => {
     });
   });
 
+  // TODO: Re-evaluate what the contract of loadProto is.
```

Comment:
Isn't it defined in a d.ts in the package? Thought I remembered seeing it there....